### PR TITLE
Add discussion about Device Owner trust of TAs

### PR DESCRIPTION
**Pull request by mingpeiwk**
Add discussion per Ben's comment to clarify whether a device owner can decide to include or reject TAs.

### CODE REVIEW

**Comment by mingpeiwk** [comment]
Issue #251

File: draft-ietf-teep-architecture.md [hunk 1]
```diff
@@ -260,7 +260,10 @@ The following terms are used:
     it is more common for the enterprise to own the device, and for any device
     user to have no or limited administration rights. In this case, the
     enterprise appoints a Device Administrator that is not the device
-    owner.
+    owner. A Device Owner may or may not have control over Untrusted Applications 
+    and TAs into the device depending on whether it is also a Device Administrator.
+    It is a Device Administrator's responsibility and privilege in general for
+    deciding to trust or reject a TA or equivalently a Trust Component Signer.
```
Comment (dthaler) [inline draft-ietf-teep-architecture.md]:
```suggestion
    deciding to trust or reject a TA or equivalently a Trusted Component Signer.
```

File: draft-ietf-teep-architecture.md [hunk 2]
```diff
@@ -260,7 +260,10 @@ The following terms are used:
     it is more common for the enterprise to own the device, and for any device
     user to have no or limited administration rights. In this case, the
     enterprise appoints a Device Administrator that is not the device
-    owner.
+    owner. A Device Owner may or may not have control over Untrusted Applications 
+    and TAs into the device depending on whether it is also a Device Administrator.
+    It is a Device Administrator's responsibility and privilege in general for
```
Comment (dthaler) [inline draft-ietf-teep-architecture.md:265]: what does "in general" mean?  Does that mean there are exceptions?   I think lines 246-255 already say this right before this paragraph so this PR should not be needed.

Comment (mingpeiwk) [reply draft-ietf-teep-architecture.md:265]: The addition is to explicitly reiterate about an Owner's control over apps. I am good to defer this for now, I will close it without commit the PR. We can revisit it if Ben insists a change later.

**Comment by mingpeiwk** [comment]
The change is considered not necessary. Will work the reviewer to agree on this as the current paragraph largely explains the role an Owner and Device Administrator has over apps.

**Pull request closed without merging**